common/lib/xmodule/xmodule/modulestore/inheritance.py

```diff
@@ -211,8 +211,8 @@ def inherit_metadata(descriptor, inherited_data):
 
 def own_metadata(module):
     """
-    Return a dictionary that contains only non-inherited field keys,
-    mapped to their serialized values
+    Return a JSON-friendly dictionary that contains only non-inherited field
+    keys, mapped to their serialized values
```

Comment:
Some more details about this "intermediate" form would be nice, since you understand it so well now. :)